pages/linux/tlp.md

```diff
@@ -0,0 +1,27 @@
+# tlp
+
+> Advanced Power Management for Linux.
+
+- Apply settings (according to the actual power source):
+
+`sudo tlp start`
+
+- Apply battery settings (_ignoring_ the actual power source):
+
+`sudo tlp bat`
+
+- Apply AC settings (_ignoring_ the actual power source):
+
+`sudo tlp ac`
+
+- Status report with configuration and all active settings:
+
+`sudo tlp-stat`
```

Comment:
This doesn't appear be the same command. Perhaps we could move these to a separate page and insert a 2nd description line that says something like this?

```
> See also `tlp-stat`.